### PR TITLE
Remove more mentions of property access

### PR DESCRIPTION
**Pull request by krsriq**
There were a couple more mentions of direct property access that are replaced with method calls in this PR.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -145,7 +142,7 @@ factory falls back to the default locale (en_US).
 // create a French faker
 $faker = Faker\Factory::create('fr_FR');
 for ($i = 0; $i < 3; $i++) {
-    echo $faker->name, "\n";
+    echo $faker->name(), "\n";
```
Comment (bramceulemans) [inline docs/index.md:145]:
```suggestion
    echo $faker->name() . "\n";
```

File: docs/index.md [hunk 2]
```diff
@@ -44,9 +44,6 @@ require_once 'vendor/autoload.php';
 
 // use the factory to create a Faker\Generator instance
 $faker = Faker\Factory::create();
-// generate data by accessing properties
```
Comment (bramceulemans) [inline docs/index.md:47]: Why'd you remove this?

File: docs/locales/pt_BR.md [hunk 3]
```diff
@@ -54,6 +54,6 @@ echo $faker->rg(false);  // '844057363'
 
 ```php
 // Generates a Brazilian formatted and valid CNPJ
-echo $faker->cnpj;        // '23.663.478/0001-24'
+echo $faker->cnpj();        // '23.663.478/0001-24'
```
Comment (GrahamCampbell) [inline docs/locales/pt_BR.md:57]: two too many space characters now

File: docs/index.md (hunk 2 above)
Comment (GrahamCampbell) [reply docs/index.md:47]: I think this comment just needs replacing, rather than being removed

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 2 above)
Comment (krsriq) [reply docs/index.md:47]: I re-added this now but changed the comment.

File: docs/locales/pt_BR.md (hunk 3 above)
Comment (krsriq) [reply docs/locales/pt_BR.md]: Thanks, fixed!

**Review by bramceulemans — APPROVED** [review]
Looks good to me now!

**Pull request merged**